### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by boevski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -614,6 +616,17 @@ Kinvey CLI is a subject to the following caveats:
 
 If problems persist, contact [Kinvey](http://support.kinvey.com).
 
+### Support for Service Environments
+
+Kinvey CLI version 4.1.0 introduces support for environments inside Flex services coinciding with the release of this feature on the backend. When upgrading from a previous Kinvey CLI version, have the following caveats in mind:
+
+- The `flex init` interactive command prompts for service environment selection in version 4.1.0 and later.
```
Comment (DennyGD) [inline README.md]: It won't prompt for a svc env if the service contains only one. Should we mention this?

File: README.md [hunk 2]
```diff
@@ -614,6 +616,17 @@ Kinvey CLI is a subject to the following caveats:
...
+- Node.js projects configured with `flex init` with a Kinvey CLI version predating 4.1.0 do not contain a service environment ID. Therefore:
+    - We recommend rerunning the `flex init` command on these projects.
+    - Flex commands targeting the current service will continue to work as long as the service has a single environment.
+    - Flex commands targeting the current service will error out if you create additional environments inside the service.
```
Comment (DennyGD) [inline README.md]: and you don't specify a svc env or is this redundant?

File: README.md (hunk 1 above)
Comment (boevski) [reply README.md]: Added.

File: README.md (hunk 2 above)
Comment (boevski) [reply README.md]: Added.

**Pull request merged**